scalableminds/webknossos
# Fix health check for public datasets

The dataset-specific health check fetches the corresponding datastore from the database. This was not allowed if not logged in, but should be (since the dataset is public)

### Steps to test:
- visit http://localhost:9000/api/datasets/sample_organization/sample_tiff/health while logged in (should be ok)
- visit it while logged out (should fail)
- set the dataset to public, visit again (should be ok even when logged out)

------
